_i18n/it/get-started/what-is-monero.md

```diff
@@ -0,0 +1,120 @@
+<div class="site-wrap">
+    <section class="container">
+            <div class="row">
+                <section class="container about-monero full col-xs-12">
+                    <div class="info-block">
+                        <div class="row center-xs">
+                            <div class="col"><h2>Quello che devi sapere</h2></div>
+                        </div>
+                        <div class="row middle-xs info-block-row private">
+                            <div class="col-lg-7 col-md-6 col-sm-8 col-xs-12  why-text">
+                                <h3>Monero è la cryptomoneta di riferimento tra quelle focalizzate in transazioni private e resistenti alla censura.</h3>
+                                <p>Molte delle cryptomonete esistenti, inclusi Bitcoin ed Ethereum, hanno una blockchain trasparente. Questo significa che le transazioni sono facilmente verificabili e tracciabili da chiunque nel mondo. Inoltre, gli indirizzi di ricezione ed invio per queste transazioni, potrebbero essere collegate all'iudentità reale di una persona. </p>
+                                <p>Monero usa la crittografia per nasconedere l'indirizzo di ricezione ed invio, così come l'ammontare della transazione.</p>
+                            </div>
+                            <div class="col-lg-5 col-md-6 col-sm-4 col-xs-12 last-sm first-xs center-xs">
+                                <img src="/img/crypto-lock.png" alt="Encrypted lock" class="main-icon">
+                            </div>
+                        </div>
+                        <div class="row middle-xs info-block-row">
+                            <div class="col-lg-5 col-md-6 col-sm-4 col-xs-12 center-xs">
+                                <img src="/img/monero-classified.png" alt="Classified document" class="main-icon">
+                            </div>
+                            <div class="col-lg-7 col-md-6 col-sm-8 col-xs-12 why-text">
+                                <h3>Le transazioni con Monero sono confidenziali ed intracciabili.</h3>
+                                <p>Ogni transazione, di default, offusca gli indirizzi di ricezione ed invio, così come l'ammontare scambiato. Essendo privato di default, ogni qualvolta un utente usa Monero rafforza la privacy di altri utenti. Questo non succede con cryptomonete selettivamente trasparenti (es. Z-Cash).</p>
+                                <p>Grazie all'offuscamento di default, Monero è fungibile, nessuna moneta può essere collegata ad una precedente transazione. Questo significa che Monero sarà sempre accettato senza rischio di censura.</p>
+                                <p>Il progetto Kovri, <a href="https://getkovri.org/">correntemente in sviluppo</a>, reindirizzerà e cripterà le transazioni attraverso nodi I2P (Invisible Internet Project). Questa tecnologia nasconderà l'indirizzo IP di chi effettua una transazione e fornisce ulteriore protezione verso un eventuale network monitoring.</p>
```

Comment:
"Il progetto Kovri, <a href="https://getkovri.org/">attualmente in sviluppo</a>, reindirizzerà e cripterà le transazioni attraverso nodi I2P (Invisible Internet Project). Questa tecnologia nasconderà l'indirizzo IP di chi effettua una transazione e fornisce ulteriore protezione verso un eventuale indesiderato monitoraggio del network."